EXILED/Exiled.Events/Patches/Events/Server/WaitingForPlayers.cs

```diff
@@ -11,12 +11,15 @@ namespace Exiled.Events.Patches.Events.Server
 
     using API.Features;
 
+    using Exiled.Events.Attributes;
+
     using HarmonyLib;
 
     /// <summary>
     ///     Patches <see cref="CharacterClassManager.Start" />.
     ///     Adds the <see cref="Handlers.Server.WaitingForPlayers" /> event.
     /// </summary>
+    [EventPatch(typeof(Handlers.Server), nameof(Handlers.Server.WaitingForPlayers))]
```

Comment:
with that added an no plugin calling this event it's would make Server.Host being null causing a lot of issue

(same as said above you can make that said in the documentation above